yt_dlp/extractor/tnaflix.py

```diff
@@ -252,7 +253,20 @@ class TNAFlixIE(TNAEMPFlixBaseIE):
             'thumbnail': r're:https?://.*\.jpg$',
             'duration': 164,
             'age_limit': 18,
-            'uploader': 'bobwhite39',
+            'categories': list,
+        },
+    }, {
+        'url': 'https://morigin.tnaflix.com/he/amateur-porn/Dirty-minded-girl-is-very-hot/video11310952',
```

Comment:
add an `only_matching` test for an `m` subdomain url too please

Reply:
removed `m` subdomain from regex- `m.tnaflix.com` now 301s to www